### DISCUSSION TITLE
Hello, does anyone know how fix `Connection has been lost issue`?

### DISCUSSION THREAD

**Discussion by SunVibing** [ORelio/Minecraft-Console-Client#2686] [Q&A]
So I have been trying to play on 2 accounts on MCC one account logins perfectly and the second account every time gets kicked for `Connection has been lost`. Anyone know a fix?
![Screenshot 2024-02-25 022821](https://github.com/MCCTeam/Minecraft-Console-Client/assets/161179802/06cc7141-a784-477f-9de2-70830ed4a27e)

**Comment by milutinke** [comment]
Try increasing the TCP timeout in settings to 60 seconds.
